resources/views/components/user-name.blade.php

```diff
@@ -19,7 +19,7 @@ class="rounded-full shadow-sm w-[25px] h-[25px]"
         />
     @endif
 
-    <a href="{{ action(\App\Http\Controllers\PublicProfileController::class, $user) }}" class="group-hover:underline">
```

Comment:
For me this didn't work, I would get the following error: 
![image](https://github.com/brendt/rfc-vote/assets/5923843/64603050-c4ac-4e7f-8e7e-3e580d841f38)

Reply:
NVM, I was missing latest migrations related to username.